### PR TITLE
Create change_brightness.py

### PR DESCRIPTION
**Pull request by furkanatesli**
### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: digital_image_processing/change_brightness.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+from PIL import Image
+
+def change_brightness(img: Image, level: float) -> Image:
+    """
+    Function to change brightness
```
Comment (cclauss) [inline digital_image_processing/change_brightness.py:5]:
```suggestion
    Change the brightness of a PIL Image to a given level.
    `level` should be a value between 0 (black) and 27 billion (oversaturated).
```
(We know it is a function... that does not need to be repeated.)
Should `level` have a default value like 1.0 or 100 or 256 or 1000?

File: digital_image_processing/change_brightness.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        """
+        return 128 + factor * (c - 128)
+
+    return img.point(brightness)
+
+if __name__ == "__main__":
+    # Load image
+    with Image.open("image_data/lena.jpg") as img:
+        # Change brightness to 170
+        cont_img = change_brightness(img, 300)
```
Comment (cclauss) [inline digital_image_processing/change_brightness.py:22]: How does 170 relate to 300?!?  If the comment is saying exactly what the code is saying then delete the comment.

**Comment by cclauss** [comment]
What is the min and max value of level?

Please add these two carriage returns.
https://travis-ci.com/github/TheAlgorithms/Python/builds/171693668#L253

**Comment by cclauss** [comment]
https://travis-ci.com/github/TheAlgorithms/Python/builds/171736313#L254

File: digital_image_processing/change_brightness.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+from PIL import Image
+
+
+def change_brightness(img: Image, level: float) -> Image:
```
Comment (cclauss) [inline digital_image_processing/change_brightness.py:4]: OK...  This looks ready to go.  Thanks for doing this.  Just a few more questions...
1. Should level be a float or an int?
2. What should be the minimum and maximum allowed values for level?
    - [ ] 0.0 to 1.0
    - [ ] -1.0 to 1.0
    - [ ] 0 to 100
    - [ ] 0 to 255
3. If level is 0 then is the resulting image black?
4. If the level is maxed then is the resulting image white or normal?

Comment (furkanatesli) [reply digital_image_processing/change_brightness.py:4]: 1. float
2. -255.0 to 255.0
3. No, if level 0 nothing changes.
4. If the level is maximized, the resulting image is white

**Pull request merged**